src/components/composer/composer/composer.ts

```diff
@@ -424,10 +429,19 @@ export class Composer extends Component<ComposerProps, SpreadsheetChildEnv> {
       .sort((a, b) => {
         return a.text.length - b.text.length || a.text.localeCompare(b.text);
       });
+
     if (searchTerm) {
-      values = fuzzyLookup(searchTerm, values, (t) => t.text);
+      values = fuzzyLookup(searchTerm, values, (t) => t.text).slice(0, AUTOCOMPLETE_ENTRIES);
     }
-    this.autoCompleteState.values = values.slice(0, 10);
+
+    this.autoCompleteState.values = values.slice(0, AUTOCOMPLETE_ENTRIES);
+    this.autoCompleteState.getHtmlContent = (value) =>
+      getHtmlContentFromPattern(
```

Comment:
why is it a function ? Can't it be set to the result of `getHtmlContentFromPattern` ?

Reply:
See https://github.com/odoo/o-spreadsheet/pull/2643#discussion_r1253082096